examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -28,47 +28,12 @@
     "</details>\n",
```

Comment:
<p>"<code>Molecule</code> objects are unusual in the computational chemistry world; they're not a thin wrapper around a SMILES, nor a collection of elements, coordinates, and formal charge" --&gt; I think I see what you're getting at but this is confusing as a first time reader. It might not be worth having this part.</p><p> </p><p>"SDF files include all a molecule's bond orders and information about aromaticity" --&gt; "SDF files include all a molecule's bond orders and formal charges"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I've changed the thin wrapper thing to be much clearer: </p><p> </p><p>"<code>Molecule</code> objects represent a collection of atoms with specified formal charges, connected by bonds with specified bond orders, optionally including any number of conformer coordinates. This is intended to closely align with a chemist's intuitive understanding of a molecule, rather than simply wrap the minimal information needed for a calculation."</p>

Reply:
Nice! Thanks.